styles/index.css

```diff
@@ -224,6 +256,11 @@ h2.font-logo {
 .slate-MentionSelect {
   @apply z-50 !important;
 }
+
+
+
+
+
```

Comment:
Was this intentional? Totally fine if it was, just curious.

Reply:
noooope, good catch!